source/api/tests/test_api_integration.py

```diff
@@ -0,0 +1,359 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+# 
+# ###############################################################################
+# PURPOSE:
+#   * Integration test for project api endpoints and workflow.
+#
+###############################################################################
+
+"""
```

Comment:
Please put USAGE info in the comment block, above, where you have the PURPOSE description.

Reply:
fixed